components/Media/MediaDetailDialog.tsx

```diff
@@ -0,0 +1,58 @@
+import Dialog from 'components/Dialog/Dialog';
+import { Media } from 'types';
+import { format } from 'date-fns';
+import s from './index.module.scss';
+
+interface Props {
+  media: Media;
+  isOpen: boolean;
+  onDismiss: () => void;
+}
+
+const humanFileSize = (bytes: number): string => {
+  if (bytes == 0) {
+    return '0.0 B';
+  }
+  const e = Math.floor(Math.log(bytes) / Math.log(1024));
```

Comment:
This seems overly complex and I think has some unexpected-ish behaviour, I'd expect `1024000` to return 1MB not 1000.0 KB for example.

I would advise moving this into a seperate utility module and testing the code.

We also don't handle negatives.